lib/pf/Portal/Profile.pm

```diff
@@ -511,6 +558,31 @@ sub findScan {
     return undef;
 }
 
+sub tiers {
+    $_[0]->{_tiers}
```

Comment:
It would be cleaner to declare self above and use it here in case we add anything to this sub